DTS-STN/Life-Journeys
# LJ-89 Side Menu

# Description

[LJ-89] Side sticky menu

created side menu 
changes to the having a child page as an examaple to be followed added for other pages.

## Test Instructions

1. Get the branch and see it sticks to the top when navigating 
1. Anchors to Open the accordion are in place but it requires extra js coding to open using the menu, this part is not implemented on this branch but as a separate task.

## Meets Definition of Done



[LJ-89]: https://lj-decd.atlassian.net/browse/LJ-89
